composer.json

```diff
@@ -75,6 +74,7 @@
         "sort-packages": true,
         "allow-plugins": {
             "pestphp/pest-plugin": true,
+            "php-http/discovery": true,
```

Comment:
When I run `composer update` composer added that line for me. I don't think it's harmful but I am unsure of the benefits. It can be edited out of the PR if necessary.

Reply:
V4 requires `php-http/discovery` so this is fine.